### DISCUSSION TITLE
Which UI libraries play most nicely with Next.js?

### DISCUSSION THREAD

**Discussion by tombburnell** [vercel/next.js#34103] [Help]
Hi
What are people having most success with? Particularly given SSR nature of Next.js and they way it already handles styles.

AntD, Material-ui, React boostrap, Tailwind-UI, Blueprint?? Others?

Thanks

**Comment by rahXephonz** [comment]
I always using JSS style which using styled-components and emotion also material UI using JSS styling too. But it would be nice if you combine them for example Material UI with TailwindCSS
